src/coreclr/vm/pefile.cpp

```diff
@@ -414,17 +414,11 @@ BOOL PEFile::Equals(PEFile *pFile)
     // because another thread beats it; the losing thread will pick up the PEAssembly in the cache.
     if (pFile->HasHostAssembly() && this->HasHostAssembly())
     {
-        UINT_PTR fileBinderId = 0;
-        if (FAILED(pFile->GetHostAssembly()->GetBinderID(&fileBinderId)))
-            return FALSE;
-
-        UINT_PTR thisBinderId = 0;
-        if (FAILED(this->GetHostAssembly()->GetBinderID(&thisBinderId)))
-            return FALSE;
+        AssemblyBinder* fileBinderId = pFile->GetHostAssembly()->GetBinder();
```

Comment:
Nit: `fileBinderId` -> `fileBinder` and also `thisBinderId` -> `thisBinder`

Reply:
Right. I missed that these are still Ids